scripts/stats-dashboard/readme-template.md

```diff
@@ -0,0 +1,99 @@
+
+The data in the corpus is delivered as TEI XML files to follow established practices. The metadata is delivered as CSV files, following a [normal form](https://en.wikipedia.org/wiki/Database_normalization) database structure while allowing for a legible git history. A more detailed description of the data and metadata structure and formats can be found in the README files in the `corpus/` folder.
+
+## Descriptive statistics at a glance
+
+Currently, we have an extensive set of Parliamentary Records (Riksdagens Protokoll) from 1867 until now. We are in the process of preparing Motions for inclusion in the corpus and other document types will follow.
+
+{sumstats_table}
+
+\* Digital original parliamentary records for some years in the 1990s are not paginated and thus do not contribute to the page count.See also §_Number of Pages in Parliamentary Records_.
+
+### Parliamentary Records over time
+
+#### Number of Parliamentary Records
+
+![Number of Parliamentary Records](scripts/stats-dashboard/figures/n-prot/n-prot.png)
+
+#### Number of Pages in Parliamentary Records
+
+![Number of Pages in Parliamentary Records](scripts/stats-dashboard/figures/prot-pages/prot-pages.png)
+
+#### Number of Speeches in Parliamentary Records
+
+![Number of Speeches in Parliamentary Records](scripts/stats-dashboard/figures/prot-speeches/prot-speeches.png)
+
+Note: We are aware of an issue whereby speeches are over counted in the data's current form in the years after 2014 -- we're working on a fix. Until then, the following static graph is a better representation of the actual speeches in the Parliamentary Records for those years.
+
+![Static Number of Speeches](scripts/stats-dashboard/figures/prot-speeches/tomas.png)
+
+#### Number of Words in Parliamentary Records
+
+![Number of Words in Parliamentary Records](scripts/stats-dashboard/figures/prot-words/prot-words.png)
+
+### Members of Parliament over time
+
+![Members of Parliament over time](scripts/stats-dashboard/figures/mp-coverage/mp-coverage.png)
+
+## Quality assessment
+
+### Speech-to-speaker mapping
+
+We check how many speakers in the parliamentary records our algorithms idenify in each release. 
+
+![Estimate of mapping accuracy](https://raw.githubusercontent.com/welfare-state-analytics/riksdagen-corpus/main/input/accuracy/version_plot.png)
+
+### Correct number of MPs over time
+
+![Ratio of MP to seats over time](scripts/stats-dashboard/figures/mp-coverage/mp-coverage-ratio.png)
+
+
+
+
+## Participate in the curation process
+
+If you find any errors, it is possible to submit corrections to them. This is documented in the [project wiki](https://github.com/welfare-state-analytics/riksdagen-corpus/wiki/Submit-corrections).
+
+
+## Acknowledgement of support
+
```

Comment:
do we want to add acknowledgement to funding agencies + project nrs. etc?

Reply:
I think that is a good idea. @fredrik1984 has both westac and swerik numbers.

Reply:
Add these two logos below. Swerik id no: IN22-0003. Westac id no: 2018-0606.

![687-riksbankens-jubileumsfond-rj](https://github.com/welfare-state-analytics/riksdagen-corpus/assets/15212990/8086cf98-ad4a-4f85-b57f-67fd461cc049)
![svart_liggande_eng](https://github.com/welfare-state-analytics/riksdagen-corpus/assets/15212990/5c1c6f94-f26d-46c5-9981-f37bba7c2d5c)